client/src/pages/Add.jsx

```diff
@@ -87,15 +87,7 @@ const Add = () => {
         )}
 
         {climbedBoulder && (
-          <>
             <h3>You have successfully saved a new entry for this boulder!</h3>
```

Comment:
I would work with a `h3` instead of an `p` element as your heading don't have any other content below that is described by your heading. :)

Reply:
I hope I understood you correctly and changed this from h3 to p.